ts/components/dialog/QuitModal.tsx

```diff
@@ -85,7 +85,10 @@ export const QuitModal = (props: SessionConfirmDialogProps) => {
           style={modalStyle}
         >
           <SpacerLG />
-          <StyledI18nSubText {...i18nMessage}></StyledI18nSubText>
+          <StyledI18nSubText
```

Comment:
Can we not do?

```tsx
<StyledI18nSubText localizerProps={i18nMessage} dataTestId="modal-description" />
```

Reply:
haha, I thought we could but that `StyledI18nSubText` is wrongly named. It's not a `styled.div` or something like this, it returns a react component

```ts
export const StyledI18nSubText = ({
  className,
  dataTestId,
  localizerProps,
}: {
  className?: string;
  dataTestId: SessionDataTestId;
  localizerProps: LocalizerComponentProps<LocalizerToken>;
}) => {
  return (
    <StyledI18nSubTextContainer className={className} data-testid={dataTestId}>
      <Localizer {...localizerProps} />
    </StyledI18nSubTextContainer>

Reply:
I'll rename it to make this more explicit